focus-android/quality/detekt.yml

```diff
@@ -72,8 +72,8 @@ comments:
     active: false
     excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/androidUnitTest/**', '**/androidInstrumentedTest/**', '**/jsTest/**', '**/iosTest/**']
   OutdatedDocumentation:
-    active: false
-    matchTypeParameters: true
+    active: true # Enabled in https://bugzilla.mozilla.org/show_bug.cgi?id=1848527
+    matchTypeParameters: false # (Default: true) Disabled in https://bugzilla.mozilla.org/show_bug.cgi?id=1848527
```

Comment:
❓ Can you explain a bit about this flag, I couldn't understand this.

Reply:
Yes, here's an example of my early rationale https://github.com/mozilla-mobile/firefox-android/pull/3885#discussion_r1342830827

Reply:
Gotcha, makes sense!